openlibrary/plugins/upstream/account.py

```diff
@@ -316,8 +319,10 @@ def POST(self):
                 return render['account/verify'](
                     username=f.username.value, email=f.email.value
                 )
-            except ValueError:
-                f.note = get_login_error('max_retries_exceeded')
+            except OLAuthenticationError as e:
+                f.note = get_login_error(e.__str__())
```

Comment:
`e.__str__()` returns the exception's arguments.  In this case, its the exception's `message`, which is a key for the `LOGIN_ERRORS` dict.